programs/editor/Tools.js

```diff
@@ -105,9 +105,12 @@ define("webodf/editor/Tools", [
                 sessionSubscribers.forEach(function (subscriber) {
                     subscriber.setEditorSession(editorSession);
                 });
-                if (formatMenuButton) {
-                    formatMenuButton.setAttribute('disabled', !editorSession);
-                }
+
+                [saveButton, saveAsButton, downloadButton, closeButton, formatMenuButton].forEach(function (button) {
```

Comment:
The open button should perhaps be fixed as well? Currently, if I supply a link to a non-existent document, then click open it throws an assertion :grin: